winml/test/collateral/models/Add_ImageNet1920WithImageMetadataBgr8_SRGB_0_1_tensor.onnx

```diff
@@ -0,0 +1,27 @@
+OnnxMLTools
+0.1.0.0000"onnxml:�
+%
+input_39
+input_40add_3Add"Addkeras_Add_ImageNet_smallZ+
+input_39
+
+
+
+�
+�2IMAGEZ+
+input_40
+
+
+
+�
+�2IMAGEb_
+add_3V
```

Comment:
for all of these onnx files, can we instead generate these on the fly? new collateral adds some level of RI complexity